src/image_emotion_gender_demo.py

```diff
@@ -203,10 +203,8 @@ def load_image(self, image_path, shape_predictor):
 
     def get_Insights(self, path_to_file):
 
-        image_path = '/Users/adelwang/Documents/Hackery/Gender-Age-Expression/GenderExpression/images/joe-biden.jpg'
-        path_to_file = '/Users/adelwang/Documents/Hackery/Gender-Age-Expression/GenderExpression/images'
-        #image_path = '../images/joe-biden.jpg'
-        #path_to_file = '../GenderExpression/images'
+        #image_path = '/Users/adelwang/Documents/Hackery/Gender-Age-Expression/GenderExpression/images/joe-biden.jpg'
```

Comment:
This can't be a local path, it needs to be a path that is passed in from mukund that represents a path to the directory in EC2